projects/1.19/assets/cloud-rider/cloudrider/lang/zh_cn.json

```
@@ -0,0 +1,37 @@
+{
+    "entity.cloudrider.cloud_rider": "云骑士",
+    "entity.cloudrider.lightning_bolt_projectile": "电闪雷鸣",
+    "entity.cloudrider.cloud_steed": "云坐骑",
+    "entity.cloudrider.electric_area_effect_cloud": "带电区域",
+   
+    "item.cloudrider.cloud_rider_spawn_egg": "云骑士刷怪蛋",
+    "item.cloudrider.gray_cloud_rider_spawn_egg": "灰云骑士刷怪蛋",
+    "item.cloudrider.white_cloud_steed": "云坐骑",
+    "item.cloudrider.gray_cloud_steed": "灰色云坐骑",
+    "item.cloudrider.misty_sponge": "云绵",
+    "item.cloudrider.foggy_sponge": "灰云绵",
+   
+    "sounds.cloudrider.entity.cloud_rider.ambient": "云骑士：咯咯笑",
+    "sounds.cloudrider.entity.cloud_rider.hurt": "云骑士：受伤",
+    "sounds.cloudrider.entity.cloud_rider.death": "云骑士：死亡",
+    "sounds.cloudrider.entity.cloud_rider.vaporize": "云骑士：蒸发",
+    "sounds.cloudrider.entity.cloud_rider.shoot": "云骑士：射击",
+    "sounds.cloudrider.entity.cloud_rider.charge": "云骑士：蓄力",
+    "sounds.cloudrider.entity.cloud_rider.charged_shot": "云骑士：释放强力电击",
+    "sounds.cloudrider.entity.cloud_rider.converted_to_gray": "云骑士：变异",
+    "sounds.cloudrider.item.sponge.fill_cloud_rider": "云骑士：被海绵吸入",
+    "sounds.cloudrider.entity.lightning_bolt_projectile.ambient": "电场嗡嗡作响",
+    "sounds.cloudrider.entity.electric_area_effect_cloud.ambient": "电缚",
+   
+    "death.attack.lightningBoltProjectile": "%1$s被%2$s击败了",
+    "death.attack.lightningBoltProjectile.item": "%1$s被%2$s使用%3$s击败了",
+    "death.attack.electrocution": "%1$s被电死了",
+    "death.attack.electrocution.player": "%1$s在试图逃离%2$s时触电身亡",
+    "death.attack.vaporize": "%1$s无影无踪地蒸发了",

```

| **审查级别**    |
|--------------|
| ⚪ 轻微 / 不阻止合并但需三思  |

原文：**%1$s was vaporized without a trace**
```suggestion
    "death.attack.vaporize": "%1$s蒸发了",
```
要么统一下面一行